lib/src/test/groovy/com/cloudbees/groovy/cps/CpsDefaultGroovyMethodsTest.groovy

```diff
@@ -268,13 +264,11 @@ class CpsDefaultGroovyMethodsTest extends AbstractGroovyCpsTest {
             ["reverseEachMapKV", "def r = ''; ['a': 1, 'b': 2, 'c': 3].reverseEach { k, v -> r += k }; return r", "cba"],
             ["reverseEachMapEntry", "def r = ''; ['a': 1, 'b': 2, 'c': 3].reverseEach { e -> r += e.key }; return r", "cba"],
 
-            /* TODO would need to translate OrderBy and ClosureComparator
             // .sort
             ["sortList", "[3, 1, -2, -4].sort { i -> i * i }", [1, -2, 3, -4]],
             ["sortArray", "([3, 1, -2, -4] as Integer[]).sort { i -> i * i }", [1, -2, 3, -4]],
-            ["sortMapEntryByKey", "[a: 3, c: 1, b: -2, d: -4].sort { e -> e.key }", [a: 3, b: -2, c: 1, d: -4]],
-            ["sortMapEntryByValue", "[a: 3, c: 1, b: -2, d: -4].sort { e -> e.value }", [d: -4, b: -2, c: 1, a: 3]],
-            */
+            ["sortMapEntryByKey", "[a: 3, c: 1, b: -2, d: -4].sort { e -> e.key }.keySet().toList()", [a: 1, b:2, c: 3, d:4].keySet().toList()],
```

Comment:
Turns out the map sort tests were passing even when they shouldn't have been - this change made them fail without the fix, so yay.